core/src/main/java/org/mskcc/cbio/oncokb/model/Query.java

```diff
@@ -16,23 +20,48 @@
 import static org.mskcc.cbio.oncokb.util.FusionUtils.FUSION_SEPARATOR;
 
 
-/**
- * TumorType generated by hbm2java
- */
+@ApiModel(description = "Enriched user annotation query")
 public class Query implements java.io.Serializable {
-    private String id; //Optional, This id is passed from request. The identifier used to distinguish the query
+    @ApiModelProperty(value = "(Nullable) The id passed in request for the user to distinguish the query.")
+    private String id;
```

Comment:
Do you know what the use case for this feature is?

Reply:
If user makes a post request with multiple queries, they may want to tag/mark queries to be able to distinguish them. I know internally, MPath uses this to identify their queries.